Makefile

```diff
@@ -1,3 +1,8 @@
+CLEANUP_FILES  = tarantool.log
+CLEANUP_FILES += *.xlog*
```

Comment:
1. A typo (`*.xlog*` -> `*.xlog`)?
2. How about `*.vylog`?

Reply:
I kept all file masks as is, but moved them to a separate variable. However 1 and 2 are good points. I think we can address it in #76. Vinyl tests are broken and I think it's a reason why `*.vylog` is not there.